libs/cvr-fixture-generator/src/generate_cvrs.ts

```diff
@@ -115,164 +120,172 @@ interface GenerateCvrsParams {
  * @param options.testMode Generate CVRs for test ballots or live ballots
  * @returns Array of generated {@link CVR.CVR}
  */
-export function* generateCvrs({
+export function generateCvrs({
   testMode,
   scannerNames,
   ballotPackage,
   includeBallotImages,
-}: GenerateCvrsParams): Generator<CVR.CVR> {
-  const { electionDefinition } = ballotPackage;
-  const { election } = electionDefinition;
-  const { ballotStyles } = election;
-  let castVoteRecordId = 0;
-  for (const ballotStyle of ballotStyles) {
-    const { precincts: precinctIds, id: ballotStyleId, partyId } = ballotStyle;
-    for (const ballotType of [
-      CVR.vxBallotType.Absentee,
-      CVR.vxBallotType.Precinct,
-    ]) {
-      for (const precinctId of precinctIds) {
-        for (const scanner of scannerNames) {
-          // Find the layout, which we'll use to determine which contests are on which page
-          const ballotPageLayouts = find(
-            ballotPackage.ballots,
-            (ballot) =>
-              ballot.ballotConfig.ballotStyleId === ballotStyleId &&
-              ballot.ballotConfig.precinctId === precinctId &&
-              ballot.ballotConfig.isLiveMode === !testMode
-          ).layout;
-          if (ballotPageLayouts.length > 2) {
-            throw new Error('only single-sheet ballots are supported');
-          }
+}: GenerateCvrsParams): IteratorPlus<CVR.CVR> {
+  return iter(
+    (function* gen() {
```

Comment:
The syntax of defining generators in TS is already a bit confusing and in this case I can see how it's a bit weirder. No complaints, I would do it the same way, but I know I would find this confusing if I stumbled upon it and I wish there were an easier syntax.

Reply:
Yeah I don't love this either. Certainly open to suggestions on how to improve this. @jonahkagan?

Reply:
I wasn't sure which approach to use. It's nice to be able to just use these methods on the result, but you're right that this is pretty 👎. I'll just return `Generator<T>`.